index.js

```diff
@@ -27,29 +38,84 @@ const urls = fs
   .split(/\r?\n/)
   .map((l) => `${l}`);
 
-const addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
+function whichBrowserIsSelected(isChromeSelected) {
+  return isChromeSelected
+    ? 'agbomdmjjfglgplpnkahbcmblehajkag'
+    : 'd56a5b99-51b6-4e83-ab23-796216679614';
+}
+let addonUUID = whichBrowserIsSelected(isChromeSelected);
 
-const options = new firefox.Options();
-options.setPreference(
-  'extensions.webextensions.uuids',
-  `{"firefox@ghostery.com": "${addonUUID}"}`,
-);
-// options.addArguments("--headless");
+let selectedBrowser = '';
+let options = ``;
+let outputPath = 'output/time';
+let driver = new Builder();
+let addon = '';
+let browserForDriver;
+
+if (isChromeSelected) {
+  selectedBrowser = 'Chrome';
+  options = new chrome.Options();
```

Comment:
no need for variable on top level scope, you can declare a `const options` here and second time in `else`

Reply:
To double the same attribute? Why. It is the same option at the end, the main difference is which browser (Firefox or Chrome) is selected because of the differences in how they are set up.